src/app/window-handler.ts

```diff
@@ -639,6 +639,13 @@ export class WindowHandler {
                     } else if (isMac && type === 'screen') {
                         const dispId = source.id.split(':')[1];
                         this.execCmd(this.screenShareIndicatorFrameUtil, [ dispId ]);
+                    } else if (isWindowsOS && type === 'screen') {
+                        logger.info('window-handler: source.display_id: ' + source.display_id);
```

Comment:
please consider using String interpolation for log statements with variable values.